uchardet-sys/Cargo.toml

```diff
@@ -18,3 +18,4 @@ libc = "*"
 
 [build-dependencies]
 pkg-config = '*'
+cmake = "*"
```

Comment:
I'm quite happy to see this dependency, especially if it helps us build on Windows and stay in sync with upstream.